docs/gaia-tutorials/join-mainnet.md

```diff
@@ -208,8 +248,4 @@ invariant broken:
 ```
 
 When submitting a invariant-broken transaction, transaction fee tokens are not
-deducted as the blockchain will halt (aka. this is a free transaction). 
-
-## Upgrade to Validator Node
-
-You now have an active full node. What's the next step? You can upgrade your full node to become a Cosmos Validator. The top 125 validators have the ability to propose new blocks to the Cosmos Hub. Continue onto [the Validator Setup](../validators/validator-setup.md).
+deducted as the blockchain will halt (aka. this is a free transaction). 
```

Comment:
```suggestion
deducted as the blockchain will halt (invariant-broken transactions are free transactions). 
```

Reply:
not sure what we mean here, what causes the blockchain to halt? (halt would happen if fees are deducted?) we can reword for clarity